Octokit/Helpers/AcceptHeaders.cs

```diff
@@ -27,7 +27,9 @@ public static class AcceptHeaders
         public const string MigrationsApiPreview = "application/vnd.github.wyandotte-preview+json";
 
         public const string ReactionsPreview = "application/vnd.github.squirrel-girl-preview";
-        
+
+        public const string SignatureVerification = "application/vnd.github.cryptographer-preview+sha";
```

Comment:
Maybe name this `SignatureVerificationPreview` just for consistency with all the other preview headers?